product_docs/docs/eprs/7/07_common_operations/10_replicating_postgres_partitioned_tables.mdx

```diff
@@ -71,7 +71,7 @@ INSERT INTO emp VALUES (7902,'FORD','ANALYST',7566,'03-DEC-81',3000,NULL,20);
 INSERT INTO emp VALUES (7934,'MILLER','CLERK',7782,'23-JAN-82',1300,NULL,10);
 ```
 
-The following creates the partitioned table in PostgreSQL or EDB Postgres Advanced Server 10 or later using declarative partitioning:
+The following creates the partitioned table in PostgreSQL or EDB Postgres Advanced Server 11 or later using declarative partitioning:
```

Comment:
@ccestes Is there a reason for this specific change?  This page contains numerous version 10 references, which were not updated.  That being said, I don't think any of them need to be updated, including possibly this one.  Although, PG and EPAS 10 may no longer be supported, I think all the references to it are still valid as they all seem to caveat it as "10 or later".